Tasks/ServiceFabricDeploy/deploy.ps1

```diff
@@ -0,0 +1,87 @@
+# For more information on the VSTS Task SDK:
+# https://github.com/Microsoft/vsts-task-lib
+
+[CmdletBinding()]
+param()
+
+Trace-VstsEnteringInvocation $MyInvocation
+try {
+    # Import the localized strings. 
+    Import-VstsLocStrings "$PSScriptRoot\task.json"
+    
+    # Load utility functions
+    . "$PSScriptRoot\utilities.ps1"
+    
+    # Collect input values
+    $publishProfilePathSearchPattern = Get-VstsInput -Name publishProfilePath -Require
+    Write-Host (Get-VstsLocString -Key SearchingForPublishProfile -ArgumentList $publishProfilePathSearchPattern) 
+    $publishProfilePath = Find-VstsFiles -LegacyPattern $publishProfilePathSearchPattern
+    Assert-SingleItem $publishProfilePath $publishProfilePathSearchPattern
+    Assert-VstsPath -LiteralPath $publishProfilePath -PathType Leaf
+    Write-Host (Get-VstsLocString -Key FoundPublishProfile -ArgumentList $publishProfilePath)
+    
+    $applicationPackagePathSearchPattern = Get-VstsInput -Name applicationPackagePath -Require
+    Write-Host (Get-VstsLocString -Key SearchingForApplicationPackage -ArgumentList $applicationPackagePathSearchPattern)
+
+    # Workaround for bug in Find-VstsFiles that doesn't handle the IncludeDirectories parameter correctly. Modifies the search pattern to look for app manifest file instead of the directory
```

Comment:
I merged this fix a couple weeks ago. Although it doesn't look like the new version was published yet. I'll follow-up on that.